lib/raygun.ts

```diff
@@ -335,6 +352,31 @@ class Raygun {
     }
   }
 
+  /**
+   * Attach as express middleware to create a breadcrumb store scope per request.
+   * e.g. `app.use(raygun.expressHandlerBreadcrumbs);`
+   * Then call to `raygun.addBreadcrumb(...)` to add breadcrumbs to the future Raygun `send` call.
+   * @param req
+   * @param res
+   * @param next
+   */
+  expressHandlerBreadcrumbs(req: Request, res: Response, next: NextFunction) {
```

Comment:
Just a general question - I obviously get that Express is very popular and commonly used, but what are the criteria for a specific 3rd party framework/tooling to get "special" consideration in a provider SDK?

Reply:
Asked internally on Slack and the response I got was to focus on ExpressJS at the moment.